RecoJets/JetProducers/plugins/CSJetProducer.h

```diff
@@ -32,13 +34,21 @@ namespace cms {
   protected:
     void runAlgorithm(edm::Event& iEvent, const edm::EventSetup& iSetup) override;
 
+    double getModulatedRhoFactor(
+        const double phi, const double eventPlane2, const double eventPlane3, const double par1, const double par2);
```

Comment:
```suggestion
    double getModulatedRhoFactor(
        double phi, double eventPlane2, double eventPlane3, double par1, double par2) const;
```
the method itself is simple enough that const modifiers to the arguments are not adding clarity